### PR TITLE
Next: enabled eslint object-shorthand rule and fixed code

### PR DESCRIPTION
**Pull request by mspae**

### CODE REVIEW

File: src/wavesurfer.js [hunk 1]
```diff
@@ -573,12 +573,12 @@ const WaveSurfer = util.extend({}, util.observer, { util }, {
         );
     },
 
-    getArrayBuffer: function (url, callback) {
+    getArrayBuffer(url, callback) {
         var my = this;
 
         var ajax = WaveSurfer.util.ajax({
-            url: url,
-            responseType: 'arraybuffer'
```
Comment (thijstriemstra) [inline src/wavesurfer.js]: The url: url was more readable imo.

**Comment by thijstriemstra** [comment]
This is already using a plugin API I'm unfamiliar with, I thought this was going to be introduced in steps?

File: src/wavesurfer.js (hunk 1 above)
Comment (mspae) [inline src/wavesurfer.js]: I changed the eslint rule to apply to methods, property shorthands are not enforced and I reverted this code to be like it was.

**Comment by mspae** [comment]
I thought the consensus was to create the next branch with all new changes and iteratively improve on the quality from there.

**Pull request merged**